src/libs/OptionsListUtils.js

```diff
@@ -186,7 +186,7 @@ function getSearchText(report, reportName, personalDetailList, isChatRoomOrPolic
     if (!isChatRoomOrPolicyExpenseChat) {
         for (let i = 0; i < personalDetailList.length; i++) {
             const personalDetail = personalDetailList[i];
-            searchTerms = searchTerms.concat([personalDetail.displayName, personalDetail.login.replace(/\./g, '')]);
+            searchTerms = searchTerms.concat([personalDetail.displayName, personalDetail.login, personalDetail.login.replace(/\.(?=[^\s@]*@)/g, '')]);
```

Comment:
Dots should be removed only from the local part of the email. Otherwise the email would like like test123@mywebsitecom instead of test123@mywebsite.com

Reply:
It does cause a problem
if you type:
test.123@mydomain.com
it will be matched for
test123@mydomaincom and it will fail

Previously if you type test.123@mydomain.com it will be converted to test123@mydomaincom and matched with test123@mydomaincom and we get a result, it's wrong but works, wrong because it treats today.news.com and todaynews.com as one domain name which is wrong

Reply:
@rushatgabhane The regex is needed either way because not only it will fix this edge case issue but it's required to get a match, as I said above:
if you type:
[test.123@mydomain.com](mailto:test.123@mydomain.com)
it will be matched for
test123@mydomaincom and it will fail

Reply:
Sorry what i meant to say is
if you type:
test123@mydomain.com
it will be matched for
test123@mydomaincom and it will fail

With the applied regex:
if you type:
test123@mydomain.com
it will be matched for
test123@mydomain.com and we will get a result

Reply:
we can but under what name?
the regex does this exactly:
remove any dots before the symbol @ as long as there are no spaces

and it does not have to be an email:
test.123@domain.com => test123@domain.com
test.123@ ==> test123@
test.123@+1516 ==> test123@+1516

Reply:
Updated